packages/wonder-blocks-dropdown/src/components/combobox.tsx

```diff
@@ -521,18 +530,44 @@ export default function Combobox({
                         updateOpenState(false);
                         handleBlur();
                     }}
-                    aria-controls={controlledWidget}
                     onKeyDown={onKeyDown}
                     aria-activedescendant={currentActiveDescendant}
                     aria-autocomplete={autoComplete}
+                    aria-controls={controlledWidget}
                     aria-expanded={openState}
+                    aria-invalid={!!error}
                     ref={comboboxRef}
                     // We don't want the browser to suggest autocompletions as
                     // the combobox is already providing suggestions.
                     autoComplete="off"
                     role="combobox"
                 />
 
+                {inputValue && !disabled && (
+                    <IconButton
+                        disabled={disabled}
+                        icon={xIcon}
+                        onClick={(e) => {
+                            e.stopPropagation();
+                            // Reset the combobox value.
+                            setInputValue("");
+                            setSelected("");
+                            onChange?.("");
+                            comboboxRef.current?.focus();
```

Comment:
question: Would it be helpful to have a screen reader announcement for when the selected value is cleared? What do you think?

Reply:
Makes total sense! I tried making this work, but the announcement logic is becoming to complex, so I'm planning to refactor it in a follow up PR (cc @marcysutton).